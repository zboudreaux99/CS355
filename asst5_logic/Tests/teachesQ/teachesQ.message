Output line 0:
  <i>expected: </i><span class='expected'>no</span> staff --- end of output --- 
  <i>observed: </i><span class='observed'>staff</span> --- end of output --- 
