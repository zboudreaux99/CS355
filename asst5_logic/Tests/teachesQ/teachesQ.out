uncaught exception: error(existence_error(procedure,teaches_section/2),teaches/0)
